commerce_coordinator/apps/commercetools/sub_messages/tasks.py

```diff
@@ -301,7 +319,10 @@ def _prepare_segment_event_properties(in_order, return_line_item_return_id):
                         properties=segment_event_properties
                     )
         else:  # pragma no cover
-            logger.debug(f'[CT-{tag}] payment intent {payment_intent_id} not refunded. Message_id: {message_id}')
+            logger.info(f'[CT-{tag}] payment intent {payment_intent_id} not refunded. Message_id: {message_id}')
```

Comment:
Should we make the 'M' lower case as we did with the [previous](https://github.com/edx/commerce-coordinator/pull/254/files#diff-b357959ac43ee7ad59ac8206c964144ecf96f2cc8d7b427d5c61674eeda885ecR287) log?